<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background:qlineargradient(spread:pad,x1:0.4,y1:0,x2:1,y2:1,stop:0.1 #FFD26F,stop:0.7 #3677FF)
}

</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="value">
    <number>13</number>
   </property>
   <property name="format">
    <string>%p</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="value">
    <number>13</number>
   </property>
   <property name="format">
    <string>%p</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="value">
    <number>13</number>
   </property>
   <property name="format">
    <string>%p</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Group 13</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
